test-native/deps.edn

```diff
@@ -0,0 +1,6 @@
+{:deps {httpkit/httpkit {:local/root ".."}}
```

Comment:
I chose deps.edn to depend on httpkit locally to be able to test changes quickly.
I didn't know how to do that with leiningen.